File: shapeview/src/main/java/cn/mtjsoft/www/shapeview/styleable/ShapeConstraintLayoutStyleable.java
package cn.mtjsoft.www.shapeview.styleable;

import cn.mtjsoft.www.shapeview.R;
import cn.mtjsoft.www.shapeview.imp.IStyleable;

/**
 * @author mtj
 * @date 2022/2/7
 * @desc
 * @email mtjsoft3@gmail.com
 */
public class ShapeConstraintLayoutStyleable implements IStyleable {
    @Override
    public int getShape() {
        return R.styleable.ShapeConstraintLayout_shape;
    }

    @Override
    public int getSolidColor() {
        return R.styleable.ShapeConstraintLayout_solidColor;
    }

    @Override
    public int getStrokeColor() {
        return R.styleable.ShapeConstraintLayout_strokeColor;
    }

    @Override
    public int getStartColor() {
        return R.styleable.ShapeConstraintLayout_startColor;
    }

    @Override
    public int getCenterColor() {
        return R.styleable.ShapeConstraintLayout_centerColor;
    }

    @Override
    public int getEndColor() {
        return R.styleable.ShapeConstraintLayout_endColor;
    }

    @Override
    public int getStartSelectColor() {
        return R.styleable.ShapeConstraintLayout_startSelectColor;
    }

    @Override
    public int getCenterSelectColor() {
        return R.styleable.ShapeConstraintLayout_centerSelectColor;
    }

    @Override
    public int getEndSelectColor() {
        return R.styleable.ShapeConstraintLayout_endSelectColor;
    }

    @Override
    public int getOrientation() {
        return R.styleable.ShapeConstraintLayout_orientation;
    }

    @Override
    public int getGradientType() {
        return R.styleable.ShapeConstraintLayout_gradientType;
    }

    @Override
    public int getGradientRadius() {
        return R.styleable.ShapeConstraintLayout_gradientRadius;
    }

    @Override
    public int getStrokeWidth() {
        return R.styleable.ShapeConstraintLayout_strokeWidth;
    }

    @Override
    public int getDashWidth() {
        return R.styleable.ShapeConstraintLayout_dashWidth;
    }

    @Override
    public int getDashGap() {
        return R.styleable.ShapeConstraintLayout_dashGap;
    }

    @Override
    public int getRadius() {
        return R.styleable.ShapeConstraintLayout_radius;
    }

    @Override
    public int getTopLeftRadius() {
        return R.styleable.ShapeConstraintLayout_topLeftRadius;
    }

    @Override
    public int getTopRightRadius() {
        return R.styleable.ShapeConstraintLayout_topRightRadius;
    }

    @Override
    public int getBottomLeftRadius() {
        return R.styleable.ShapeConstraintLayout_bottomLeftRadius;
    }

    @Override
    public int getBottomRightRadius() {
        return R.styleable.ShapeConstraintLayout_bottomRightRadius;
    }

    @Override
    public int isOpenSelector() {
        return 0;
    }

    @Override
    public int getTextNormalColor() {
        return 0;
    }

    @Override
    public int getTextSelectColor() {
        return 0;
    }

    @Override
    public int getSolidSelectColor() {
        return 0;
    }

    @Override
    public int getStrokeSelectColor() {
        return 0;
    }
}

File: shapeview/src/main/java/cn/mtjsoft/www/shapeview/styleable/ShapeConstraintLayoutStyleable.kt
package cn.mtjsoft.www.shapeview.styleable

import cn.mtjsoft.www.shapeview.imp.IStyleable
import cn.mtjsoft.www.shapeview.R

/**
 * @author mtj
 * @date 2022/2/7
 * @desc
 * @email mtjsoft3@gmail.com
 */
class ShapeConstraintLayoutStyleable : IStyleable {
    override val shape: Int
        get() = R.styleable.ShapeConstraintLayout_shape
    override val solidColor: Int
        get() = R.styleable.ShapeConstraintLayout_solidColor
    override val strokeColor: Int
        get() = R.styleable.ShapeConstraintLayout_strokeColor
    override val startColor: Int
        get() = R.styleable.ShapeConstraintLayout_startColor
    override val centerColor: Int
        get() = R.styleable.ShapeConstraintLayout_centerColor
    override val endColor: Int
        get() = R.styleable.ShapeConstraintLayout_endColor
    override val startSelectColor: Int
        get() = R.styleable.ShapeConstraintLayout_startSelectColor
    override val centerSelectColor: Int
        get() = R.styleable.ShapeConstraintLayout_centerSelectColor
    override val endSelectColor: Int
        get() = R.styleable.ShapeConstraintLayout_endSelectColor
    override val orientation: Int
        get() = R.styleable.ShapeConstraintLayout_orientation
    override val gradientType: Int
        get() = R.styleable.ShapeConstraintLayout_gradientType
    override val gradientRadius: Int
        get() = R.styleable.ShapeConstraintLayout_gradientRadius
    override val strokeWidth: Int
        get() = R.styleable.ShapeConstraintLayout_strokeWidth
    override val dashWidth: Int
        get() = R.styleable.ShapeConstraintLayout_dashWidth
    override val dashGap: Int
        get() = R.styleable.ShapeConstraintLayout_dashGap
    override val radius: Int
        get() = R.styleable.ShapeConstraintLayout_radius
    override val topLeftRadius: Int
        get() = R.styleable.ShapeConstraintLayout_topLeftRadius
    override val topRightRadius: Int
        get() = R.styleable.ShapeConstraintLayout_topRightRadius
    override val bottomLeftRadius: Int
        get() = R.styleable.ShapeConstraintLayout_bottomLeftRadius
    override val bottomRightRadius: Int
        get() = R.styleable.ShapeConstraintLayout_bottomRightRadius
    override val isOpenSelector: Int
        get() = 0
    override val textNormalColor: Int
        get() = 0
    override val textSelectColor: Int
        get() = 0
    override val solidSelectColor: Int
        get() = 0
    override val strokeSelectColor: Int
        get() = 0
}